packages/augur-tools/src/templates-lists.ts

```diff
@@ -8113,6 +8113,95 @@ export const LIST_VALUES = {
   ],
 };
 
+export const FIN_EXCHANGES_CLOSING_OFFSETS = {
+  'New York Stock Exchange (traded in USD)': {
+    offset: -4,
+    hour: 16,
+    minutes: 0,
+  },
+  'Nasdaq (traded in USD)': { offset: -5, hour: 16, minutes: 0 },
+  'American Stock Exchange (traded in USD)': {
+    offset: -4,
+    hour: 16,
+    minutes: 0,
+  },
+  'Tokyo Stock Exchange (traded in Japanese Yen)': {
+    offset: 9,
+    hour: 15,
+    minutes: 0,
+  },
+  'London Stock Exchange (traded in Pound Sterling)': {
+    offset: 0,
+    hour: 16,
+    minutes: 30,
+  },
+  'Shanghai Stock Exchange (A shares traded in Yuan)': {
+    offset: 8,
+    hour: 15,
+    minutes: 0,
+  },
+  'Shanghai Stock Exchange (B shares traded in USD)': {
+    offset: 8,
+    hour: 15,
+    minutes: 0,
+  },
+  'Hong Kong Stock Exchange (traded in HKD Hong Kong Dollar)': {
+    offset: 8,
+    hour: 16,
+    minutes: 0,
+  },
+  'Toronto Stock Exchange (traded in Canadian Dollar)': {
+    offset: -4,
+    hour: 16,
+    minutes: 0,
+  },
+  'Euronext NV (traded in Euro)': { offset: 2, hour: 17, minutes: 30 },
+};
+
+export const FIN_INDEXES_CLOSING_OFFSETS = {
```

Comment:
do we need a ```FIN_INDEXES_HOILDAYS_OFFSETS```

Reply:
I could combine them but they are different. Exchanges and Indexes even though indexes are traded on exchanges. I would like to keep them separate